### ISSUE TITLE
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-YsrkRl/rogues/

### ISSUE THREAD

**Issue by HYChou0515** [macd/rogues#6]
Using `pip install rogues` to install, error occurs:
```
Collecting rogues
  Using cached https://files.pythonhosted.org/packages/1a/75/eae551e084510737ba792fabbf3ed5d31c7627e3db2d92d9a63a641ad4b1/rogues-0.4.0.tar.gz
    Complete output from command python setup.py egg_info:
    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.34.tar.gz
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-C5awKW/rogues/setup.py", line 4, in <module>
        use_setuptools()
      File "distribute_setup.py", line 152, in use_setuptools
        return _do_download(version, download_base, to_dir, download_delay)
      File "distribute_setup.py", line 131, in _do_download
        to_dir, download_delay)
      File "distribute_setup.py", line 201, in download_setuptools
        src = urlopen(url)
      File "/mnt/home1/hychou/anaconda2/lib/python2.7/urllib2.py", line 154, in urlopen
        return opener.open(url, data, timeout)
      File "/mnt/home1/hychou/anaconda2/lib/python2.7/urllib2.py", line 435, in open
        response = meth(req, response)
      File "/mnt/home1/hychou/anaconda2/lib/python2.7/urllib2.py", line 548, in http_response
        'http', request, response, code, msg, hdrs)
      File "/mnt/home1/hychou/anaconda2/lib/python2.7/urllib2.py", line 473, in error
        return self._call_chain(*args)
      File "/mnt/home1/hychou/anaconda2/lib/python2.7/urllib2.py", line 407, in _call_chain
        result = func(*args)
      File "/mnt/home1/hychou/anaconda2/lib/python2.7/urllib2.py", line 556, in http_error_default
        raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
    urllib2.HTTPError: HTTP Error 403: SSL is required
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-C5awKW/rogues/
```
--
python version: 2.7.15 :: Anaconda, Inc.
pip version:  18.0

having upgrade the pip setuptools: `pip install --upgrade setuptools`

**Comment by macd** [comment]
Haven't used 2.7 in several years, but will look at it over the next week or so.   If you need something sooner, you might want to clone the repo.  It's all straightforward Python & Numpy. It pip installs OK on Python 3.6, BTW.

**Comment by macd** [comment]
This is now fixed in rogues 0.5.0, which also fixes a handful of data type errors as well.  Tested on Python 2.7.15, 3.6.5 and 3.7.0
